packages/babel-helpers/src/helpers/assertThisInitialized.ts

```diff
@@ -1,6 +1,6 @@
 /* @minVersion 7.0.0-beta.0 */
 
-export default function _assertThisInitialized(self) {
+export default function _assertThisInitialized<T>(self: T): T {
```

Comment:
```suggestion
export default function _assertThisInitialized<T>(self: T | undefined): T {
```

`_assertThisInitialized` does accept `undefined` though.